fe/fe-core/src/main/java/org/apache/doris/nereids/trees/plans/commands/insert/OlapInsertExecutor.java

```diff
@@ -183,6 +183,7 @@ protected void beforeExec() {
 
     @Override
     protected void onComplete() throws UserException {
+        boolean isMow = table instanceof OlapTable && ((OlapTable) table).getEnableUniqueKeyMergeOnWrite();
```

Comment:
it's better to enable it only for cloud mow, not all mow table

Reply:
@zhannngchen done